docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1021,7 +1126,63 @@ The following table lists all the search category attributes the server can rece
 
 > **Note**
 >
-> These Attributes support Tizen 5.0 and Higher for Mobile and Wearable.
+> These attributes support Tizen 5.0 and Higher for Mobile and Wearable.
+
+## Media Controller Display Mode Attributes
+
+The following table lists all the display mode attributes that the client can receive and send the command to:
+
+**Table: Media controller display mode attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Display modes**                |                                          |
+| `MC_DISPLAY_MODE_LETTER_BOX`     | Display mode is letter box               |
+| `MC_DISPLAY_MODE_ORIGIN_SIZE`    | Display mode is origin size              |
+| `MC_DISPLAY_MODE_FULL_SCREEN`    | Display mode is fullscreen               |
+| `MC_DISPLAY_MODE_CROPPED_FULL`   | Display mode is cropped fullscreen       |
+
+> **Note**
+>
+> These attributes support Tizen 5.5 and Higher for Mobile and Wearable.
+
+## Media Controller Display Rotation Attributes
+
+The following table lists all the display rotation attributes that the client can receive and send the command to:
+
+**Table: Media controller display rotation attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Display rotations**            |                                          |
+| `MC_DISPLAY_ROTATION_NONE`       | Display is not rotated                   |
+| `MC_DISPLAY_ROTATION_90`         | Display is rotated 90 degrees            |
+| `MC_DISPLAY_ROTATION_180`        | Display is rotated 180 degrees           |
+| `MC_DISPLAY_ROTATION_270`        | Display is rotated 270 degrees           |
+
+> **Note**
+>
+> These attributes support Tizen 5.5 and Higher for Mobile and Wearable.
+
+## Media Controller Ability Attributes
+
+The following table lists all the search category attributes that the server can receive:
+
+**Table: Media controller ability attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Ability**                      |                                          |
+| `MC_ABILITY_SHUFFLE`             | Ability for shuffle                      |
+| `MC_ABILITY_REPEAT`              | Ability for repeat                       |
+| `MC_ABILITY_PLAYBACK_POSITION`   | Ability for playback position            |
+| `MC_ABILITY_PLAYLIST`            | Ability for playlist                     |
+| `MC_ABILITY_CLIENT_CUSTOM`       | Ability for custom event                 |
+| `MC_ABILITY_SEARCH`              | Ability for search                       |
+
+> **Note**
+>
+> These Attributes support Tizen 5.5 and Higher for Mobile and Wearable.
```

Comment:
My bad, I missed it. 
Much thanks to @mijin85cho for pointing it out. 
@haejeongkim 
Kindly change Attributes to attributes. 
> These attributes support Tizen 5.5 and Higher for Mobile and Wearable.

Reply:
Done